### ISSUE TITLE
numbers of calsses

### ISSUE THREAD

**Issue by Jinjicheng** [AntiAegis/Overcoming-Catastrophic-Forgetting#3]
i find the task A and task B are all 10 classification. if task A is 10 classification, and task B is 5 classification, then how to compute EWC?

**Comment by thuyngch** [comment]
Hi @Jinjicheng,
In my opinion, we can compute FIM for all parameters, excepting ones in the last layer (logits).
